internal/encryption/keyprovider/openbao/config.go

```diff
@@ -0,0 +1,97 @@
+package openbao
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	openbao "github.com/openbao/openbao/api"
+	"github.com/opentofu/opentofu/internal/encryption/keyprovider"
+)
+
+type Config struct {
+	Address string `hcl:"address,optional"`
+	Token   string `hcl:"token,optional"`
+
+	KeyName        string `hcl:"key_name"`
+	DataKeyBitSize int    `hcl:"data_key_bit_size,optional"`
```

Comment:
Let's keep this consistent as key_length to match the other key providers.

Reply:
Thanks for the comment! Originally, I aimed to match OpenBao docs, but it could confuse the end user. Changed `data_key_bit_size` (bits) to `key_length` (bytes) in 4d09d69182154da558bd7aa116ea4dd6feade7f3.